src/bosh-director/lib/bosh/director/api/controllers/orphan_disks_controller.rb

```diff
@@ -0,0 +1,13 @@
+require 'bosh/director/api/controllers/base_controller'
+
+module Bosh::Director
+  module Api::Controllers
+    class OrphanDisksController < BaseController
+      get '/' do
+        content_type(:json)
+        orphan_json = OrphanDiskManager.new(@logger).list_orphan_disks
```

Comment:
Traditionally we try to create these managers once in the `initialize`, then in the function invoke the desired method ([example](https://github.com/barthy1/bosh/blob/c85667afea2e1e12b06240542243a3d49f68004a/src/bosh-director/lib/bosh/director/api/controllers/deployments_controller.rb#L43)). Not critical, but helps us keep our `new`ers in a consistent place until we can do better at DI.